components/table/table-wrapper.js

```diff
@@ -370,9 +375,19 @@ export class TableWrapper extends RtlMixin(PageableMixin(SelectionMixin(LitEleme
 		this._tableScrollers = {};
 	}
 
+	connectedCallback() {
+		super.connectedCallback();
+
+		this.addEventListener('d2l-dropdown-open', this._handlePopoverOpen);
```

Comment:
I chose this `Popover` naming because eventually we want to have a Popover element or mixin that both dropdowns and tooltips can use, in which case the event handling here will accommodate both. 

I chose not to introduce the new `d2l-popover-open` and `d2l-popover-close` events yet. When that work happens, we'll com back in here and update the events its listening for.